### ISSUE TITLE
Updates to OLC Guides page not appearing on frontend for certain users 

### ISSUE THREAD

**Issue by JesseCCSI** [NRGI/resourcecontracts.org#1036]
We published a new training module on OLC today and made other edits the Guides page (openlandcontracts.org/guides). The edits appear on the frontend when viewed using a PC (Chrome Version 55.0.2883.87 m) - see screenshot (which shows how the page should now look) - but not when using a Mac (Chrome Version 55.0.2883.95 64 bit - up to date). 

We're announcing the launch of the training module tomorrow afternoon EST, so would be helpful if this could be addressed asap. 

(cc @KaitlinCCSI) 

**This is how section 2 of the page should appear:**
![image](https://cloud.githubusercontent.com/assets/14910764/22091788/cf178a0c-ddc6-11e6-8316-f1c879225bd5.png)

**Comment by manishgs** [comment]
It is due to browser cache. Please clear browser cache and try again.

**Comment by JesseCCSI** [comment]
Thanks @manishgs but both @KaitlinCCSI and I cleared our browser cache and are still having this problem on both our Macs. 

Might something else be the cause?

**Comment by SamCCSI** [comment]
Reopening to address @JesseCCSI 's comment

**Comment by manishgs** [comment]
@JesseCCSI @SamCCSI 
There are two containers/instances running to serve subsite, one container has updated content and another has cached content which was not flushed yet.  So for some user's browser showing old content were due to serving from cached container. For now we have cleared the cache on both containers. I've talked with **Vitamin** regarding the issue. They will work on it to resolve the issue.

cc @charlesyoung
